modules/windows-machineset-nutanix.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc
+
+[id="windows-machineset-nutanix_{context}"]
+= Sample YAML for a Windows MachineSet object on Nutanix
+
+This sample YAML defines a Windows `MachineSet` object running on Nutanix that the Windows Machine Config Operator (WMCO) can react upon.
+
+[source,yaml]
+----
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  labels:
+    machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+  name: <infrastructure_id>-windows-worker-<zone> <2>
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+      machine.openshift.io/cluster-api-machineset: <infrastructure_id>-windows-worker-<zone> <2>
+  template:
+    metadata:
+      labels:
+        machine.openshift.io/cluster-api-cluster: <infrastructure_id> <1>
+        machine.openshift.io/cluster-api-machine-role: worker
+        machine.openshift.io/cluster-api-machine-type: worker
+        machine.openshift.io/cluster-api-machineset: <infrastructure_id>-windows-worker-<zone> <2>
+        machine.openshift.io/os-id: Windows <3>
+    spec:
+      metadata:
+        labels:
+          node-role.kubernetes.io/worker: "" <4>
+      providerSpec:
+        value:
+          apiVersion: machine.openshift.io/v1
+          bootType: "" <5>
+          categories: null
+          cluster: <6>
+            type: uuid
+            uuid: <cluster_uuid>
+          credentialsSecret:
+            name: nutanix-credentials <7>
+          image: <8>
+            name: <cluster_id>
+            type: name
+          kind: NutanixMachineProviderConfig <9>
+          memorySize: 16Gi <10>
+          metadata:
```

Comment:
I don't think this metadata field is needed, I can't find it in our template, nor in developers docs https://github.com/openshift/windows-machine-config-operator/blob/master/docs/machineset-nutanix.md